src/Symfony/Component/Translation/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+5.1.0
+-----
+ * Support `name` attribute on `unit` element from xliff2 to be used as a translation key instead of always the `source` element.
```

Comment:
for consistency with existing lines:

missing blank line before
should started with a verb, past form
should end with no dot